Mathlib/Topology/ContinuousMap/Bounded/Normed.lean

```diff
@@ -67,6 +67,9 @@ lemma neg_norm_le_apply (f : α →ᵇ ℝ) (x : α) :
 lemma apply_le_norm (f : α →ᵇ ℝ) (x : α) :
     f x ≤ ‖f‖ := (abs_le.mp (norm_coe_le_norm f x)).2
 
+lemma abs_apply_le_norm (f : α →ᵇ ℝ) (x : α) :
+    |f x| ≤ ‖f‖ := norm_coe_le_norm f x
+
```

Comment:
Do definitely want this lemma?

Surely we don't want to duplicate every lemma that mentions `norm` with a variant spelled using `abs` for the case that the relevant space is `ℝ`?